kibbeh/src/modules/room/UserPreviewModal.tsx

```diff
@@ -225,9 +235,9 @@ const UserPreview: React.FC<{
 };
 
 export const UserPreviewModal: React.FC<JoinRoomAndGetInfoResponse> = ({
-  room,
-  users,
-}) => {
+                                                                         room,
```

Comment:
what is this?

Reply:
probably some weird quirk of the WebStorm formatter, I'll look around in the settings

Reply:
fixed